app/src/App.tsx

```diff
@@ -169,7 +169,9 @@ const AppWithRouter = (props) => {
   const { componentsData } = props;
   return (
     <Router>
-      <App componentsData={componentsData} />
+      <CountProvider>
```

Comment:
We'll also have a surrounding element here introduced by @petro97's PR to display notifications. Might want to sync up with him to ensure there's no conflicts with these elements.